_wallets/unstoppable.md

```diff
@@ -0,0 +1,46 @@
+---
+# This file is licensed under the MIT License (MIT) available on
+# http://opensource.org/licenses/MIT.
+
+id: unstoppable
+title: "Unstoppable Wallet"
+titleshort: "Unstoppable"
+compat: "mobile android ios"
+level: 2
+platform:
+  - mobile:
+    name: mobile
+    os:
+      - name: android
+        text: "walletunstoppable"
+        link: "https://play.google.com/store/apps/details?id=io.horizontalsystems.bankwallet"
+        source: "https://github.com/horizontalsystems/unstoppable-wallet-android"      
+        screenshot: "unstoppable.png"
+        check:
+          control: "checkgoodcontrolfull"
+          validation: "checkpassvalidationspvp2p"
+          transparency: "checkpasstransparencyopensource"
+          environment: "checkpassenvironmentmobile"
+          privacy: "checkpassprivacybasic"
+          fees: "checkpassfeecontroldynamic"
+        privacycheck:
+          privacyaddressreuse: "checkpassprivacyaddressrotation"
+          privacydisclosure: "checkfailprivacydisclosurespv"
+          privacynetwork: "checkfailprivacynetworknosupporttor"
+      - name: ios
+        text: "walletunstoppable"
+        link: "https://itunes.apple.com/app/bank-bitcoin-wallet/id1447619907?ls=1&mt=8"
```

Comment:
Coin also currently uses ?mt=8 so that part is probably not tripping our syntax checker.  None of our other entries use ls=1 so perhaps it is the &?  Those parameters tell iOS what type of media it is (8 is an iOS app) so that they know what app to open (and perhaps skip the web page in between).  The other parameter is also related to skipping the web page.